Sources/Swift/Core/Tools/SentryLogSwift.swift

```diff
@@ -11,7 +11,7 @@ import Foundation
 /// and the thread sanitizer would surface these race conditions. We accept these race conditions for
 /// the log messages in the tests over adding locking for all log messages.
 @objc
-@_spi(Private) public class SentryLog: NSObject {
+@_spi(Private) public class SentryLogSwift: NSObject {
```

Comment:
`h`: In my opinion naming the internal SDK debug logger a `...Swift` type is misleading. We have some cases where we have an Objective-C wrapper around a Swift type, and vice-versa.

Instead I would like to propose something like `SentrySDKLog` or `SentryDiagnosticLogger` or such.

cc @philipphofmann in case you have inputs for the naming.

Reply:
Went with `SentrySDKLog` for now.